provider/cluster/kube/apply.go

```diff
@@ -73,3 +77,33 @@ func applyIngress(kc kubernetes.Interface, b *ingressBuilder) error {
 	}
 	return err
 }
+
+func applyLeaseNS(kc kubernetes.Interface) error {
+	obj, err := kc.CoreV1().Namespaces().Get(manifestNamespace, metav1.GetOptions{})
```

Comment:
ineffectual assignment to `obj`

Reply:
> 'Cause I'm a twenty-first century digital boy
I don't know how to live but I got a lot of toys
My daddy's a lazy middle-class intellectual
My mommy's on Valium, so **ineffectual**
Ain't life a mystery, yeah?

https://www.youtube.com/watch?v=Mw3E9lUXoKw

Reply:
accurate